pkg/utils/configs/manifest.go

```diff
@@ -42,14 +43,14 @@ func (manager *ManifestManager) GetDefaultManifestWithEnv() Manifest {
 	}
 }
 
-func getSkupperConfiguredImages(components []string, enableSHA bool) SkupperManifest {
+func getSkupperConfiguredImages(components []string, enableSHA bool, runningPods map[string]string) SkupperManifest {
 	var manifest SkupperManifest
 
 	for _, component := range components {
 		var image SkupperComponent
 		image.Component = component
-		image.Version = images.GetImageVersion(component)
-		image.Images = images.GetImages(component, enableSHA)
+		image.Version = images.GetImageVersion(component, runningPods)
+		image.Images = images.GetImages(component, enableSHA, runningPods)
```

Comment:
Minor personal preference: I think it would be cleaner to do a two stage lookup here, checking the pods first then falling back to image utils. That way the image utils could be left to do its one thing, i.e. determine the configured images.

Reply:
done